db/post_migrate/20201111005205_move_data_to_event_dates.rb

```diff
@@ -51,9 +51,12 @@ def due_reminders(event)
     return [] if event.reminders.blank?
     event.reminders.split(",").map do |reminder|
       value, unit = reminder.split('.')
-      date = event.original_starts_at - value.to_i.send(unit)
+
+      allowed = ["years", "months", "weeks", "days", "hours", "minutes", "seconds"]
+      next if allowed.include?(unit)
```

Comment:
```suggestion
      next if !allowed.include?(unit)
```
(my bad, the logic in my suggestion was backwards)